frontend/src/components/LoginForm.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { login } from '../reducers/user';
+import { SignUpForm } from './SignUpForm';
+import { Button } from './Button';
+import '../styles/style.css';
+
+export const LoginForm = () => {
+  const dispatch = useDispatch();
+  const errorMessage = useSelector(store => store.user.login.errorMessage);
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [isLogin, setIsLogin] = useState(true);
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    dispatch(login(email, password));
+    setEmail('');
+    setPassword('');
+  };
+
+  const handleSetLogin = event => {
+    event.preventDefault();
+    setIsLogin(false);
+  };
+
+  return (
+    <main className="main-container">
+      {!isLogin ? (
+        <SignUpForm />
```

Comment:
We are wondering about the structure here. The component "LoginForm" is the one rendering the SignUp-component. Shouldn't the parent for SignUp be the same parent as for LoginForm?